ext/compatibility.h

```diff
@@ -113,6 +113,26 @@ static zend_always_inline bool zend_parse_arg_obj(zval *arg, zend_object **dest,
 
 #define Z_PARAM_OBJ_OR_NULL(dest) \
     Z_PARAM_OBJ_EX(dest, 1, 0)
+    
```

Comment:
Linter complains about white spaces in this line, sadly GitHub suggestions wont allow me to keep an empty line, it would only allow to remove the line or have two empty lines 😉